config/config_test.go

```diff
@@ -34,6 +34,11 @@ auth-username: myuser
 auth-password: mypasswd
 identity-location: localhost:18082
 identity-public-key: 0000
+identity-api-url: "http://example.com/identity"
+identity-api-username: idmuser
+identity-api-password: idmpass
```

Comment:
I added those because they were there before, but missing from the test. I've now removed all traces of passwords.